services/core-api/app/__init__.py

```diff
@@ -6,7 +6,7 @@
 from flask_cors import CORS
 from flask_restplus import Resource, apidoc
 from sqlalchemy.exc import SQLAlchemyError
-from flask_jwt_oidc.exceptions import AuthError
+from app.flask_jwt_oidc_local.exceptions import AuthError
```

Comment:
we are switching to the local version of this library, with enhancements. 

This is the main change to support multiple integration sso clients. 

https://github.com/thorwolpert/flask-jwt-oidc library has been built to work with only one single JWT validator and client. 

`app.flas_jwt_oidc_local` is a copy of the main jwtManager() class from the above library, and it is extended to be initiated with JWT env vars. More expln. in that section.